Warning for Blender-glTF-export: models containing blend shapes should Export Deformation Bones Only (#6001)
godotengine/godot-docs: tutorials/assets_pipeline/importing_scenes.rst

changes in a text based format. The second is you need the texture files separate from the material file. If you don't need
either of those glTF binary files are fine.

> .. warning::
> 
>     If your model contains blend shapes (aka "shape keys" and "morph targets"),
>     your glTF export setting ``Export Deformation Bones Only`` needs to be configured to **Enabled**
>     under the Animation export configurations.
> 
>     Exporting non-deforming bones anyway will lead to incorrect shading in GLES3.

Reviewer: ```suggestion
.. warning::

    If your model contains blend shapes (also known as "shape keys" and "morph targets"),
    your glTF export setting **Export Deformation Bones Only** needs to be configured to **Enabled**
    under the Animation export configurations.

    Exporting non-deforming bones anyway will lead to incorrect shading in GLES3.
```
